profiles/tests/test_gql_profile_query.py

```diff
@@ -276,7 +278,10 @@ def test_staff_user_needs_required_permission_to_access_verified_personal_inform
         )
     )
 
-    executed = user_gql_client.execute(query, service=service)
+    token_payload = {"loa": "high" if has_loa else "low"}
```

Comment:
We could maybe test with "substantial" (or both), because I guess the loa won't be "high" in majority of the cases.

Reply:
Yep. Although this isn't the main "loa" test, it's just here for checking that the "loa" doesn't let you access others' profiles.

Reply:
Actually parametrizing the test by the "has_loa" argument is useless. The "loa" claim can always be included and it will cover all cases worth testing (in this test).

Reply:
True. Didn't notice that this was about the staff user.

Reply:
Changed the "loa" to "substantantial" and removed the parametrization, i.e. the "loa" claim is always present in this particular test.